test/unit/lib/DataSourceDefinition/Mysql.test.ts

```diff
@@ -68,7 +68,7 @@ suite("DataSourceDefinition/Mysql @remote", () => {
       await new Mysql({ host: "x" }).connectionTest();
       assert.fail("connectionTest does not fail");
     } catch (err) {
-      assert.ok(/getaddrinfo ENOTFOUND/.test(err.message));
+      assert.ok(/getaddrinfo (ENOTFOUND|EAI_AGAIN)/.test(err.message));
```

Comment:
In GitHub Actions environment, error code is EAI_AGAIN.
https://github.com/mtgto/bdash/runs/2045120780?check_suite_focus=true